### DISCUSSION TITLE
Developer pack verification ID

### DISCUSSION THREAD

**Discussion by NamanK125** [community/community#110589] [GitHub Education]
I had no 2 FA.Now I did. But it's saying uploaded proof already used. What to do?

**Comment by samus-aran** [comment]
Hi @NamanK125 

We would recommend your take another live capture of your evidence to resolve this warning 👍

**Comment by NamanK125** [comment]
Is it not clear?

**Reply by samus-aran** [reply]
Sorry, what is not clear? 

My answer was to address this message you said you received: **saying uploaded proof already used** 🙂

**Comment by NamanK125** [comment]
I understand now. Thank you for your response.😊
